c_glib/arrow-glib/compute.h

```diff
@@ -675,6 +675,40 @@ GArrowRoundToMultipleOptions *
 garrow_round_to_multiple_options_new(void);
 
 
+/**
+ * GArrowUtf8NormalizeForm:
```

Comment:
Could you use `UTF` not `Utf`? We don't want to capitalize acronym such as HTTP and CSV.

```suggestion
 * GArrowUtf8NormalizeForm:
```